src/components/Modal/Modal.md

```diff
@@ -0,0 +1,79 @@
+
+The container makes no assumptions about the inner content and can be styled to fit whatever implementation is needed.
+
+## Examples
+
+### Basic Usage
+Activating the modal is done through passing a `boolean` flag as the `showModal` `prop`. This gives the parent the ability to set if the modal will render open or closed on initial render.
+
+Toggling the state of the modal is also handled by passing a callback through the `toggle` `prop`. The callback is a `voided` function that accepts a  `boolean` as whether the modal is visible. When either the `escape` key is hit, or outside of the modal content is clicked, the modal will pass a boolean value of `false` to the `toggle` method.
+
+
+```jsx
+  import React, { useState } from 'react'
+
+  function ModalExample() {
+    const [isOpen, setIsOpen] = useState(false)
+
+    return (
+      <>
+        <button onClick={() => setIsOpen(true)}>Open Modal</button>
+        <Modal showModal={isOpen} toggle={setIsOpen}>
+          <div style={{ background: '#fff', padding: '16px 24px', width: '400px' }}>
+            <h1>You're a Wizard Harry!</h1>
+            <p>
+              Mobilicorpus lumos aparecium impedimenta nox serpensortia flagrate. Tarantallegra totalus levicorpus aparecium aresto immobilus. Inflamarae leviosa protego aresto imperio impedimenta unction locomotor alohomora locomotor. Me specialis impedimenta patronum wingardium mortis impedimenta evanesco immobilus finite.
+            </p>
+            <button type="button" onClick={() => setIsOpen(false)}>Okay</button>
+          </div>
+        </Modal>
+      </>
+    )
+  }
+
+  <ModalExample />
+```
+
+### Setting `aria-describedby` and `aria-labelledby`
+
+Since the modal makes no assumptions about the content being rendered to the outlet, setting the `aria-describedby` and `aria-labelledby` properties are left up to the implementation.  The `<Modal />` does accept to optional properties, `props.ariaDescribedBy` and `props.ariaLabelledBy` which will set their respective `aria` properties on the inner modal to whatever `id` selector is passed to them.
+
+The example below also demonstrates how a takeover style modal could be rendered:
+
+```jsx
+  import React, { useState } from 'react'
+
+  function ModalExample() {
+    const [isOpen, setIsOpen] = useState(false)
+
+    const HEADER_ID = 'long-modal-heading'
+    const DESC_ID = 'long-modal'
+
+    return (
+      <>
+        <button onClick={() => setIsOpen(true)}>Open Modal</button>
+        <Modal showModal={isOpen} toggle={setIsOpen} ariaLabelledBy={HEADER_ID} ariaDescribedBy={DESC_ID}>
+          <div style={{ background: '#fff', padding: '16px 24px', overflow: 'scroll', width: '100vw', height: '100vh' }}>
```

Comment:
You can import css into the styleguide (see `src/nora/components/Example/Example.module.scss` for example) and I'd prefer we don't encourage inline styling if possible

Reply:
👍